modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserForm.tsx

```diff
@@ -56,12 +60,30 @@ const UserForm = () => {
 		formState: {errors},
 		handleSubmit,
 		register,
+		setValue,
+		watch,
 	} = useForm<UserFormDefault>({
-		defaultValues: userAccount,
+		defaultValues: {...userAccount, roles: []},
 		resolver: yupResolver(yupSchema.user),
 	});
 
+	const rolesWatch = watch('roles') as number[];
+
 	const _onSubmit = (form: UserFormDefault) => {
+		let hasError = false;
+
+		if (!rolesWatch?.length) {
+			hasError = true;
+
+			Liferay.Util.openToast({
+				message: i18n.translate('please-select-one-or-more-roles'),
+				type: 'danger',
+			});
+		}
+		if (hasError) {
+			return false;
```

Comment:
No need to return false

```suggestion
			return;
```